src/Node.ts

```diff
@@ -2660,7 +2660,7 @@ export abstract class Node<Config extends NodeConfig = NodeConfig> {
   rotation: GetSet<number, this>;
   zIndex: GetSet<number, this>;
 
-  scale: GetSet<Vector2d | undefined, this>;
+  scale: GetSet<Vector2d, this>;
```

Comment:
Either this `GetSet` type is incorrect for the `NodeConfig.scale` type is incorrect. I assume it's this one.

Reply:
Looks like it is not correct, I don't think `scale` can return undefined.